module/data/actor/npc.mjs

```diff
@@ -410,6 +410,18 @@ export default class NPCData extends CreatureTemplate {
     const context = await this._prepareEmbedContext();
     const section = document.createElement("section");
     section.innerHTML = await renderTemplate("systems/dnd5e/templates/actors/embeds/npc-embed.hbs", context);
+
+    /**
+     * A hook event that fires after an embedded NPC stat block rendered.
+     * @function dnd5e.renderNPCStatBlock
+     * @memberof hookEvents
+     * @param {Actor5e} actor                   NPC being embedded.
+     * @param {HTMLCollection} contents         HTML elements to be inserted.
+     * @param {DocumentHTMLEmbedConfig} config  Configuration for embedding behavior.
+     * @param {EnrichmentOptions} options       Original enrichment options.
+     */
+    Hooks.call("dnd5e.renderNPCStatBlock", this.parent, section.children, config, options);
```

Comment:
Is passing `section.children` useful? You can manipulate the elements and delete them, but you can't insert any I don't think. Most render hooks give you the root element, so it's probably fine to do that here too.

Reply:
My one concern is that the root element isn't actually inserted as part of the embed, so it might cause people to try and add classes whatnot to that element which wouldn't do anything.

Reply:
Should we create a `template` element instead maybe? That should hopefully not imply that setting classes on it is meaningful. Alternatively we could convert it to an array before sending it to the hook, allow the hook to mutate it, and then call `section.replaceChildren`, but that's extra performance cost when there's no guarantee a hook will even be called.

I just thought that being able to insert elements is something an API consumer may want to do, but I don't feel strongly so I've marked it as approved and you can go with whatever you feel is best.

Reply:
Yeah, it looks like the enrichment system is fine accepting a `DocumentFragment`, so I can just use a template and return `template.contents`.